__tests__/search.js

```diff
@@ -39,15 +40,17 @@ class MockVFSServiceProvider extends ServiceProvider {
 describe('Search', () => {
   let core;
   let search;
-
+  let options = {};
+  options = MySearchAdapter;
```

Comment:
This is wrong and not working. The API you added to the search class is `options = { adapters: [] }`